.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,24 @@
 ---
-name: 'Pull Request: New issue'
-about: Report and propose a fix for a (small) issue in Hazel
+name: 'Pull Request'
+about: Propose a fix implementation for a (small) issue in Hazel
```

Comment:
Propose a fix **for** implementation for a (small) issue in Hazel

Reply:
Maybe just
```suggestion
about: Propose a fix for a (small) issue in Hazel
```
or
```suggestion
about: Propose a fix or implementation for a (small) issue in Hazel
```
or even
```suggestion
about: Propose a fix/implementation for a (small) issue in Hazel
```
I think having the double "for" and even just including "implementation" is more wordy than this needs to be.